.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 10
+    target-branch: "unstable"
+    groups:
+      all-dependencies:
+        applies-to: version-updates
+        patterns:
+          - "*"
```

Comment:
Can you explain why we did this here? It seems that `groups` isn't obligatory. I imagine you are doing it due to `version-updates`, which my search pointed out doesn't consider "minor" releases like bug fixes. If I am right, maybe we can make the update unrestricted and see if it works.